### PR TITLE
Add dev calls to contributing guidelines

### PR DESCRIPTION
**Pull request by KirstieJane**
Addresses (but does not complete #270)

Changes proposed in this pull request:

- Add dev calls & calendar to contributing guidelines

Happy to move this to a different part if that would be better? Let me know what you think's best 😸

### CODE REVIEW

**Comment by jbteves** [comment]
@KirstieJane I'm slightly confused; how is this addressing #254 ? Or is this to loop the users on that thread in?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,6 +27,14 @@ We also maintain a [gitter chat room][link_gitter] for more informal conversatio
 There is significant cross-talk between these two spaces, and we look forward to hearing from you in either venue!
 As a reminder, we expect all contributions to `tedana` to adhere to our [code of conduct][link_coc].
 
+### Monthly developer calls
+
+We run monthly developer calls via Zoom.
+You can see the schedule via the `tedana` [google calendar](https://calendar.google.com/calendar/embed?src=pl6vb4t9fck3k6mdo2mok53iss%40group.calendar.google.com&ctz=Europe%2FLondon).
+An agenda will be circulated in the gitter channel in advance of the meeting.
```
Comment (jbteves) [inline CONTRIBUTING.md]: Hm, I feel that if we say one will be circulated, we should say how far in advance. We should probably also decide a few more details on how the agenda will be managed, but that's out of the scope of this PR.

**Comment by KirstieJane** [comment]
Sorry for the wrong issue link! 🤦🏼‍♀️ Fixed now ✨

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,6 +27,14 @@ We also maintain a [gitter chat room][link_gitter] for more informal conversatio
 There is significant cross-talk between these two spaces, and we look forward to hearing from you in either venue!
 As a reminder, we expect all contributions to `tedana` to adhere to our [code of conduct][link_coc].
 
+### Monthly developer calls
+
+We run monthly developer calls via Zoom.
+You can see the schedule via the `tedana` [google calendar](https://calendar.google.com/calendar/embed?src=pl6vb4t9fck3k6mdo2mok53iss%40group.calendar.google.com&ctz=Europe%2FLondon).
```
Comment (emdupre) [inline CONTRIBUTING.md]:
```suggestion
You can see the schedule via the `tedana` [google calendar](https://calendar.google.com/calendar/embed?src=pl6vb4t9fck3k6mdo2mok53iss%40group.calendar.google.com).
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (KirstieJane) [reply CONTRIBUTING.md]: Yeaaaaah.... I feel like that would be nice..... but difficult to commit to at the moment...

Comment (jbteves) [reply CONTRIBUTING.md]: Fair enough; looks great!

**Comment by KirstieJane** [comment]
Ah - I'm not an authorised user 🙀 

Ah well, please can someone merge? 👾

**Comment by emdupre** [comment]
Whaaaa ? I'll look into this.

**Pull request merged**